src/test/resources/application.properties

```diff
@@ -0,0 +1,5 @@
+spring.datasource.url=jdbc:h2://mem;DB_CLOSE_DELAY=-1
```

Comment:
don't we already have application.yml?
How does spring differentiate this config from application.yml config?

Reply:
Because its src/test not main